components/post_view/post_reaction/post_reaction.tsx

```diff
@@ -13,13 +13,18 @@ import {Emoji} from 'mattermost-redux/types/emojis';
 
 import {Locations} from 'utils/constants';
 import {localizeMessage} from 'utils/utils.jsx';
+import {makeGetUniqueReactionsToPost} from 'utils/post_utils';
 
 import OverlayTrigger from 'components/overlay_trigger';
 import Tooltip from 'components/tooltip';
 import ChannelPermissionGate from 'components/permissions_gates/channel_permission_gate';
 import EmojiIcon from 'components/widgets/icons/emoji_icon';
 import EmojiPickerOverlay from 'components/emoji_picker/emoji_picker_overlay.jsx';
 
+import store from 'stores/redux_store.jsx';
```

Comment:
Generally, we shouldn't be accessing the global instance of the redux store from most places. Since this is in a component, we'd generally want to pass in any other data we need using its `mapDispatchToProps`, but since we're only using this when handling a click, I think I have a better idea (see below)